README.md

```diff
@@ -555,20 +554,8 @@ $pagerfantaHtml = $myView2->render($pagerfanta, $routeGenerator);
 $pagerfantaHtml = $myView2->render($pagerfanta, $routeGenerator, array('next_message' => 'Siguiente!!'));
 ```
 
-## Todo
-
-## Author
-
-Pablo Díez - <pablodip@gmail.com>
-
-## License
```

Comment:
I don't think removing the License section is the right approach. Although since you're here, I'd change this to: `Pagerfanta is licensed under the [MIT License](/LICENSE.md)` or similar